source/extensions/dynamic_modules/sdk/rust/build.rs

```diff
@@ -2,16 +2,30 @@ use std::env;
 use std::path::PathBuf;
 
 fn main() {
-    println!("cargo:rerun-if-changed=abi.h");
-    let bindings = bindgen::Builder::default()
-        .header("../../abi.h")
-        .header("../../abi_version.h")
-        .parse_callbacks(Box::new(bindgen::CargoCallbacks::new()))
-        .generate()
-        .expect("Unable to generate bindings");
+  // This is Envoy CI specific: Check if "/opt/llvm/bin/clang" exists, and if it does, set the
+  // CLANG_PATH environment variable. CLANG_PATH is for clang-sys used by bindgen:
+  // https://github.com/KyleMayes/clang-sys?tab=readme-ov-file#environment-variables
+  //
+  // "/opt/llvm/bin/clang" exists in Envoy CI containers. If the clang doesn't exist there, bindgen
+  // will try to use the system clang from PATH. So, this doesn't affect the local builds.
+  // In any case, clang must be found to build the bindings.
+  //
+  // TODO: add /opt/llvm/bin to PATH in the CI containers. That would be a better solution.
```

Comment:
@phlax can i go ahead and add `/opt/llvm/bin` to the PATH env of CI container? Then we can delete this special (a bit hacky) code. wdyt?

Reply:
nvm, I think this is more about bypassing PATH to the clang actions or not -